webapp/pages/admin/categories.vue

```diff
@@ -31,7 +31,7 @@ export default {
   },
   apollo: {
     Category: {
-      query: gql(`
+      query: gql`
```

Comment:
:open_mouth: code creep!!!

Reply:
What means `code creep`?
I couldn’t find a understandable translation or explanation … @mattwr18

Reply:
Oh sorry, it's when changes are made that are not directly related to the scope of the issue/Pull Request